File: examples/python/automations/mailboxes/delete.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.mailboxes.delete(
    mailbox_id="mb_FRf6FX5fYkenZ_JJlL5GD",
)

File: cookbook/node/automations/mailboxes/delete.js
import { config } from 'dotenv';
import { Retab } from '@retab/node';

config({ path: '../../../.env.production' });

const reclient = new Retab();

const automation = await reclient.processors.automations.mailboxes.delete({
  mailbox_id: "mb_FRf6FX5fYkenZ_JJlL5GD",
});

console.log(JSON.stringify(automation, null, 2));